### ISSUE TITLE
Online players not shown

### ISSUE THREAD

**Issue by larsl-net** [dirien/minecraft-prometheus-exporter#488]
### What happened?

The metric `minecraft_player_online_total` does not show any names in FTB Infinity Evolved. For other metrics the name is correct, here replaced with `PLAYERNAME`.

Minecraft: `1.7.10`
Forge jar: `forge-1.7.10-10.13.4.1614-1.7.10-universal.jar`

```bash
I have no name!@b94c6c8c-d68e-432c-8a23-57ec35d0690b:~$ curl -s 127.0.0.1:9150/metrics | grep "online"
# HELP minecraft_player_online_total Players currently online (1 if player is online)
# TYPE minecraft_player_online_total counter
minecraft_player_online_total{player="0"} 1
minecraft_player_online_total{player="["} 1
minecraft_player_online_total{player="]"} 1
I have no name!@b94c6c8c-d68e-432c-8a23-57ec35d0690b:~$ curl -s 127.0.0.1:9150/metrics | grep "player_stat"
# HELP minecraft_player_stat_total Statistic related to the player: xp, current_xp, food_level, health, score, advancements
# TYPE minecraft_player_stat_total gauge
minecraft_player_stat_total{player="PLAYERNAME",stat="current_xp"} 0
minecraft_player_stat_total{player="PLAYERNAME",stat="food_level"} 5
minecraft_player_stat_total{player="PLAYERNAME",stat="health"} 2
minecraft_player_stat_total{player="PLAYERNAME",stat="score"} 10
minecraft_player_stat_total{player="PLAYERNAME",stat="xp"} 10
```

### How can we reproduce this?

Create a docker-compose.yaml
```yaml
version: '3.5'

services:
  minecraft:
    image: ghcr.io/pterodactyl/yolks:java_8
    ports:
      - 25565:25565
      - 9150:9150
    volumes:
      - ./data:/data
    entrypoint: bash /data/entrypoint.sh
```

Install FTB Infinity Evolved server
```bash
cd data
curl https://api.modpacks.ch/public/modpack/23/99/server/linux -Lo server_installer
chmod +x server_installer
./server_installer --auto

curl https://github.com/dirien/minecraft-prometheus-exporter/releases/download/v0.17.1/minecraft-exporter_0.17.1.linux-amd64.tar.gz -Lo minecraft-exporter.tar.gz
tar xzf minecraft-exporter.tar.gz
chmod +x minecraft-exporter
```

Create custom start script `data/entrypoint.sh` after execute `chmod +x data/entrypoint.sh`
```sh
cd /data
echo "Start Exporter"
./minecraft-exporter --mc.rcon-address=:8003 --mc.rcon-password=Test123 --mc.world=./world &

echo "Start Server"
#echo "java -Xms128M -Xmx${1}M -jar ${2}"
"jre/jdk8u312-b07-jre/bin/java" -javaagent:log4jfix/Log4jPatcher-1.0.0.jar -XX:+UseG1GC -XX:+UnlockExperimentalVMOptions -Xmx4096M -Xms4096M -jar forge-1.7.10-10.13.4.1614-1.7.10-universal.jar nogui
```

Enable rcon in `server.properties`
```properties
enable-rcon=true
rcon.port=8003
rcon.password=Test123
```

Start server
```bash
docker compose up -d
```

### minecraft-exporter --version

```bash
minecraft_exporter, version 0.17.1 (branch: HEAD, revision: 64b3d8074d199bb92097d3401055de34a8cc3990)
  build user:       goreleaser
  build date:       2022-11-24T08:02:35Z
  go version:       go1.19.2
  platform:         linux/amd64
```

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dirien** [comment]
Hi @larsl-net,

Thanks for creating the issue! I will try this out with the `FTB Infinity Evolved` modpack.

Sometimes, there are mods who change the output of some commands and my parser just dies! So this kind of issues help me to build a more robust exporter!

**Comment by dirien** [comment]
HI @larsl-net,

I checked now the reported issue and it looks like that I cant fix it. The RCON command command to list the current players comes always back with `Players currently online: [ 0 ]`. While I could fix the parsing error, it will still not help as the list of players is not present.

See for the bug report and possible solution: https://forum.feed-the-beast.com/threads/rcon-list-command.155769/

I tried it with vanilla forge and the `FTB Infinity Evolved` modpack and can confirm the issue on the link above.

Vanilla Forge 1.7.10:

![image](https://user-images.githubusercontent.com/38325136/210238951-ae94648d-40bc-4ebf-ac7c-71a09dd77fdc.png)

`FTB Infinity Evolved`:
![image](https://user-images.githubusercontent.com/38325136/210238984-d644b080-2c29-44f5-b528-e991048bfd37.png)

Sorry :(
